docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -195,6 +197,92 @@ command completion, or environment variables are presented throughout the
 Ansible Collection Testing because the act of installing the stable release of
 Ansible containing `ansible-test` is expected to setup those things for you.
 
+.. _meta_runtime_yml:
+
+meta directory
+--------------
+
+A collection can store some additional runtime metadata in a ``runtime.yml`` file in the ``meta`` directory.
+
+The ``runtime.yml`` file supports the top level keys:
+
+- *plugin_routing*:
+
+  A mapping of content which has been deprecated or removed or moved to a different location.
+
+  .. code:: yaml
+
+     plugin_routing:
+       inventory:
+         my_inventory:
+           redirect: another_namespace.collection_name.inventory_plugin
+       modules:
+         my_module:
+           deprecation:
+             removal_date: "2020-11-30"
+             warning_text: my_module will be removed in a future release of this collection. Use new_module instead.
+           redirect: another_namespace.collection_name.module_name
+       module_utils:
+         my_util:
+           redirect: namepsace.collection_name.sub_folder.util_name
+
+- *requires_ansible*:
+
+  The collection's version constraint for Ansible.
+
+  .. code:: yaml
+
+     requires_ansible: ">=2.10"
+
+- *import_redirection*:
+
+  A mapping of names for Python import statements for resources which have moved.
+
+  .. code:: yaml
+
+     import_redirection:
+       ansible.module_utils.old_utility:
+         redirect: ansible_collections.collection_name.plugins.module_utils.new_location
+
+- *action_groups*:
+
+  A mapping of group names and the modules they contain. Action group names allow playbooks to cut down on boilerplate by configuring a common module setting (such as authentication for the AWS modules) via module_defaults instead of per task.
+
+  .. code:: yaml
+
+     action_groups:
+       group_name:
+         - my_module:
+       another_namespace.collection.group_name:
+         - another_module:
+
+  In a playbook the collection's group name should be prefixed with ``group/`` to identify it as a group rather than an action/module.
+
+  .. code:: yaml
+
+     module_defaults:
+       group/my_collection.my_namespace.group_name:
+         common_opt: value
+       group/another_namespace.collection.group_name:
+         another_common_opt: value
+
+- *action_groups_redirection*:
+
+  A mapping of action group names and redirected action group names to support shorter names and sharing common groups between collections.
+
+  .. code:: yaml
+
+     action_groups_redirection:
+       group_name:
+         redirect: common_group
```

Comment:
Here is the documentation I'm referring to.

Reply:
The documentation was correct, code was wrong